Review Caesar Cipher

becca6223 opened the pull request:
thanks for reviewing!

becca6223 committed:
Delete README.md

becca6223 committed:
Add files via upload

These files are for Caesar Cipher

dsarpang commented on caesar.c:
```diff
@@ -0,0 +1,120 @@
+{
+	Input stringInfo;
+	//Get number of shifts
+	stringInfo.numShift = atoi(strtok(string, ":"));
```
what is string is null? Also, avoid choosing a variable name that is commonly a reserved word in most languages

becca6223 replied:
Do you mean what if the string is NULL?

dsarpang commented on caesar.c:
```diff
@@ -0,0 +1,120 @@
+	stringInfo.numShift = atoi(strtok(string, ":"));
+	
+	//Get the actual Input string and malloc it 
+	char* actstring = strtok(NULL, "\0");
```
Prefer verbose variable names or use common abbreviations. I.e. "actualString" is more readable

dsarpang commented on caesar.c:
```diff
@@ -0,0 +1,120 @@
+	}
+}
+
+void _EncodeChar(char* add_char, int shiftDirect, int modValue, int numShift, int arraySize)
```
Nice job creating a helper function for this!

dsarpang reviewed (changes requested):
We'll cover some of the review in class. My main feedback I see is that some of the variable names are a bit confusing (remember the 'readable' aspect of good code we learned in week1). Try to think more about what meaningful names are and I'll review another iteration. Nice work though!

becca6223 committed:
Update variable names in caesar.c

change variable name to be readable, meaningful. 
And don't choose variable names that are used in programming languages.
